static/docs/commands-reference/diff.md

```diff
@@ -0,0 +1,44 @@
+# diff
+
+Show changes between files or directories that are under DVC control.
+
+## Synopsis
+
+```usage
+  usage: dvc diff [-h] [-q | -v] [-t TARGET] a_ref [b_ref]
+
+  positional arguments:
+    a_ref                 Git reference from which diff calculates
+    b_ref                 Git reference until which diff calculates,
+                          if omitted diff shows the difference
+                          between current HEAD and a_ref
+```
+
+## Description
+
+Given two Git commit references (commit hash, branch or tag name, etc) `a_ref`
+and `b_ref`, this command shows a standard textual comparisson (or "diff") of
+all the files under DVC control among both refs. This is similar to how `git
+diff` works.
+
+If the `-t` option is used, the diff is limited to the `TARGET` file or
```

Comment:
Must the target be under DVC control?

Reply:
Yes I think so. In my tests it gives you ```$ dvc diff -t new-file HEAD^
ERROR: failed to get 'diff new-file HEAD^' - Have not found file/directory 'new-file' in the commits
``` otherwise.